### PR TITLE
Change to load the json.gz file

### PR DESCRIPTION
**Pull request by SvenDowideit**

### CODE REVIEW

**Comment by SvenDowideit** [comment]
follow on of @unclejack 's change. One more thing is needed before merge :) the release.sh script needs to set the Content-type (I'll do it after i get back from dinner with my parents)

**Comment by SvenDowideit** [comment]
@fredlf @jamtur01 this makes a pretty massive difference to the docs site responsiveness. I'll update the 1.14 docs update PR once we merge this so we can get it out there asap.

There are a couple more refinements that I'll work on over the next week or so, as we're still running with `async: false` (I need to dive into the tipuesearch code and make it more async), and also need to fix up some kind of fallback / build for `make docs`, as searching there needs some help. but those are further refinements :) the gzip and moving the tipuesearch initialisation to later makes for a pretty huge improvement on my end already.

@mariusGundersen - can you confirm if http://dowideit-docs.s3-website-ap-southeast-2.amazonaws.com/ is better for your side of the world? I have not looked deeply enought into tipuesearch to get rid of the `async: false` - thats one of the next steps.

File: docs/release.sh [hunk 1]
```diff
@@ -88,7 +91,6 @@ upload_current_documentation() {
 	echo "  to $dst"
 	echo
 	#s3cmd --recursive --follow-symlinks --preserve --acl-public sync "$src" "$dst"
```
Comment (jamtur01) [inline docs/release.sh]: Remove this too?

**Comment by jamtur01** [comment]
LGTM

File: docs/release.sh [hunk 2]
```diff
@@ -100,6 +102,9 @@ upload_current_documentation() {
 		echo "$run"
 		echo "======================="
 		$run
+
+	# Make sure the search_content.json.gz file has the right content-encoding
+	aws s3 cp --profile $BUCKET --cache-control "max-age=3600" --content-encoding="gzip" --acl public-read "site/search_content.json.gz" "$dst"
```
Comment (jfrazelle) [inline docs/release.sh]: can you use the variable so it will work for the master docs :D

**Comment by fredlf** [comment]
LGTM. Thanks for handling this, @SvenDowideit!

**Pull request merged**